cutlass/cf.go

```diff
@@ -356,23 +358,81 @@ func (a *App) PushNoStart() error {
 	return nil
 }
 
-func (a *App) V3Push() error {
-	if err := a.PushNoStart(); err != nil {
+func (a *App) V3CreateApp() error {
+	args := []string{"v3-create-app", a.Name}
+	command := exec.Command("cf", args...)
+	command.Stdout = DefaultStdoutStderr
+	command.Stderr = DefaultStdoutStderr
+	if err := command.Run(); err != nil {
 		return err
 	}
+	return nil
+}
 
-	args := []string{"v3-push", a.Name, "-p", a.Path}
-	if len(a.Buildpacks) > 1 {
-		for _, buildpack := range a.Buildpacks {
-			args = append(args, "-b", buildpack)
+func (a *App) V3ApplyManifest() error {
+	// load a.Manifest, modify app name, save tmp file and use that
+	if a.Manifest == "" {
+		return fmt.Errorf("Must set a.Manifest to path of manifest")
+	}
+	apps, err := manifest.ReadAndInterpolateManifest(a.Manifest, nil, nil)
+	if err != nil {
+		return err
+	}
+
+	// modify appname to generated test name
+	app := apps[0]
+	app.Name = a.Name
+
+	tmpManifestFile, err := ioutil.TempFile(os.TempDir(), "manifest-*.yml")
```

Comment:
Use RandString (https://github.com/cloudfoundry/libbuildpack/blob/f7fe6466a8efe94915a0933a340ff087972ff8cf/util.go#L239-L246) to make it a more unique name, without a wildcard